### DISCUSSION TITLE
fluentbit syslog output plugin

### DISCUSSION THREAD

**Discussion by sara-hann** [fluent/fluent-bit#5968] [General]
Hello, I see this documentation on the fluentbit syslog output plugin https://docs.fluentbit.io/manual/pipeline/outputs/syslog/, but the plugin is not listed here with the other fluentbit output plugins https://docs.fluentbit.io/manual/v/1.3/output. I am trying to locate instructions for using the syslog output plugin. I also see that it's not listed in the output plugin list on the GitHub Repo README https://github.com/fluent/fluent-bit/blob/master/README.md but it is a out_syslog folder within the repo under the plugins folder. No README for it though. I'm confused. Is this plugin still available? if so, where can I find documentation on using it? thank you so much :)

**Comment by agup006** [comment]
Hi @sara-hann the syslog output plugin is available in the docs you referenced and available in the latest version (1.9.8) as of this post. The older docs you listed are for an older version where the plugin is not available

The following goes over how to use it with the latest version: https://docs.fluentbit.io/manual/pipeline/outputs/syslog/
